sdk/communication/azure-communication-chat/azure/communication/chat/_chat_thread_client.py

```diff
@@ -246,8 +246,8 @@ def send_message(
         :keyword str sender_display_name: The display name of the message sender. This property is used to
           populate sender name for push notifications.
         :keyword callable cls: A custom type or function that will be passed the direct response
-        :return: SendChatMessageResult, or the result of cls(response)
-        :rtype: ~azure.communication.chat.SendChatMessageResult
+        :return: str, or the result of cls(response)
+        :rtype: str
```

Comment:
This change should not have been made - could we fix it before the next release goes out?
Instead we should use something like this:
`return { 'id': send_chat_message_result.id }`

Reply:
Will change accordingly in the next iteration